variables.tf

```diff
@@ -1,9 +1,52 @@
-########################################################################################################################
+##############################################################################
 # Input Variables
-########################################################################################################################
+##############################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "cluster_id" {
+  type        = string
+  description = "Id of the target IBM Cloud OpenShift Cluster"
+  nullable    = false
+}
+
+variable "operator_helm_release_namespace" {
+  type        = string
+  description = "Namespace to deploy the helm releases. Default to ibm-mq-operator helm release"
+  default     = "ibm-mq-operator"
+  nullable    = false
+}
+
+variable "add_ibm_operator_catalog" {
+  type        = bool
+  description = "Flag to configure the IBM Operator Catalog in the cluster before installing the IBM MQ Operator. Default is true"
+  default     = true
+}
+
+variable "create_ibm_mq_operator_namespace" {
+  type        = bool
+  description = "Flag to create the namespace where to deploy the IBM MQ Operator. Default to false"
```

Comment:
I would update with `Whether to create the namespace where the IBM MQ operator is deployed. Default is `false`.`